src/jobTree/jobStores/awsJobStore.py

```diff
@@ -76,7 +76,8 @@ def __init__( self, region, namePrefix, config=None ):
         self.jobs = self._getOrCreateDomain( 'jobs', create )
         self.versions = self._getOrCreateDomain( 'versions', create )
         self.files = self._getOrCreateBucket( 'files', create, versioning=True )
-        self.stats = self._getOrCreateBucket( 'stats', create )
+        self.stats = self._getOrCreateBucket( 'stats', create, versioning=True )
+        self.bucketMap = {'files':self.files, 'stats':self.stats}
```

Comment:
The bucket map is a way to get the bucket object itself given the name. After retrieving a files'  attributes from simpleDB, including the bucketName, I still need a reference to the actually bucket object. I could do a switch if(bucketName=='files') bucket=self.files, but I this is easier and more flexible if we decide to add a bucket in the future.